x-pack/test/api_integration/apis/endpoint/alerts.ts

```diff
@@ -49,7 +65,7 @@ export default function({ getService }: FtrProviderContext) {
         expect(body.alerts.length).to.eql(defaultPageSize);
         expect(body.request_page_size).to.eql(defaultPageSize);
         /**
-         * No page_index was specified. It should return page 0.
+         * No page_index was speciied. It should return page 0.
```

Comment:
why don't you like the letter 'f'

Reply:
@marshallmain Must have been careless vimming.